package.json

```diff
@@ -28,8 +28,8 @@
     "postinstall": "git submodule update --init && npm run fetch-data",
     "semantic-release": "semantic-release",
     "travis-deploy-once": "travis-deploy-once",
-    "bundlesize": "bundlesize",
-    "size": "npm run reset-cache && npm run build && npm run bundlesize",
+    "size-build": "npm run reset-cache && npm run build",
+    "size-limit": "size-limit",
```

Comment:
why not to make `size-build` a part of `size-limit` script? it does not seem to be functional without it

Reply:
https://github.com/andresz1/size-limit-action#usage